spec/models/user_spec.rb

```diff
@@ -334,6 +334,25 @@
       end
     end
 
+    context "when name contains HTML tags" do
+      it "sanitizes HTML tags from name" do
+        user = build(:user, name: "<script>alert('xss')</script>Test name")
+        user.save!(validate: false)
+        expect(user.display_name).to eq "alert('xss')Test name"
+      end
+
+      it "prevents saving names with dangerous HTML tags" do
+        user = build(:user, name: "<a href='https://malicious.com'>Click me</a>")
+        expect(user.valid?).to eq(false)
+        expect(user.errors[:name]).to include("cannot contain HTML tags")
+      end
+
+      it "allows names with non-dangerous content" do
```

Comment:
Please add a test case of `Alive & Bob` too since we have many names with `&` in them.